src/main/java/de/tum/in/www1/artemis/domain/GradeType.java

```diff
@@ -0,0 +1,5 @@
+package de.tum.in.www1.artemis.domain;
+
+public enum GradeType {
```

Comment:
I am not totally sure what the difference here is. What effect does each of them have? Pls add a clarifying comment here 👍🏻

Reply:
It's not a significant difference, more like an accessory to describe whether the scale is being used for regular grading (the grade steps contain your typical grade names - 1.0, 1.3...) or if it would map percentages to bonus points for a course